api/envoy/http/backend_routing/config.proto

```diff
@@ -34,7 +34,7 @@ message BackendRoutingRule {
   }
 
   PathTranslation path_translation = 4
-      [(validate.rules).enum = { not_in: [ 0 ] }];
+      [(validate.rules).enum = { in: [ 1, 2 ] }];
```

Comment:
not_in 0 is the same as in [1,2].   I don't see the different.  Why do we need this change?

Reply:
The warning here does a good job explaining it: https://developers.google.com/protocol-buffers/docs/reference/cpp-generated#enum

> Proto 3 enums are open enum types with possible values outside the range of specified symbols.

`not_in 0` will allow any other number, including numbers outside the range [1, 2].

In the corpus example I added, you can see `path_translation: 1197`. Our filter factory currently accepts this.

Reply:
I see.  Thanks